spec/jobs/document_proofing_job_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe DocumentProofingJob, type: :job do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
😂

Reply:
Roasted by the rails generator, I'll add specs to the jobs

Reply:
Added specs to ensure a proofing job would be able to run successfully in 44d19c6, though there was more setup than I hoped (especially for document proofing)